src/IMU.h

```diff
@@ -33,6 +33,11 @@ class IMUClass {
   int begin();
   void end();
 
+  //interrupt management
+  void attachInterrupt(voidFuncPtr callback); // Enable interrupt.
```

Comment:
Is there only one type of interrupt the IMU supports?

Is there a way to configure the thresholds?

Reply:
yes there are other interrupt, yes there is a way to configure the threshold.
This was done for a demonstration in this way.